ApollosProject/apollos-apps
# Fix: change route to actually work.

## ✏️ Description
This PR just changes the route so that it works as expected when you click on the daily habit hero card for daily scripture. 

## 🔬 To Test
1. pull this branch 
2. run the app and open daily. 
3. use hero card to navigate to daily habits. 
4. see that all navigation works. 

## 📸 Screenshots


## ⚠️ To-do Before Merge
